<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LowDiskSpaceDialog</class>
 <widget class="QDialog" name="LowDiskSpaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Low disk space</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{ font-family: &quot;SF Pro&quot;; font-size: 14px; font-weight: 400; }

QPushButton {
  padding-top: 4px; padding-bottom: 4px; padding-left: 14px; padding-right: 14px;
  border-style: solid; border-width: 1px; border-color: #d7d6d5; border-radius: 6px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lExplanation">
       <property name="text">
        <string notr="true">Not enough space label</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lDriveIcon">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lDiskName">
           <property name="styleSheet">
            <string notr="true">font: 700</string>
           </property>
           <property name="text">
            <string notr="true">Local Disk</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lFreeSpace">
           <property name="text">
            <string notr="true">Free space:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lTotalSize">
           <property name="text">
            <string notr="true">Total size:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bCancel">
           <property name="styleSheet">
            <string notr="true">QPushButton {
  background-color : #FCFCFC;
  color : #000000;
}
QPushButton:hover { background-color: #fdfdfd; }
QPushButton:pressed { background-color: #cacaca; }</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bTryAgain">
           <property name="styleSheet">
            <string notr="true">QPushButton {
  background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #4B91F7, stop:1 #367AF6);
  color : #FFFFFF;
}
QPushButton:hover { background-color: #5e95f8; }
QPushButton:pressed { background-color: #0a58e6; }</string>
           </property>
           <property name="text">
            <string>Try Again</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
